test/decl/func/functions_uninhabited_param.swift

```diff
@@ -7,4 +7,4 @@ func foo(baz: Never) -> Int { // expected-note {{the function body will never be
   return 0
 }
 
-func bar(baz: Never) -> Int {} // ok
+func bar(baz: Never) -> Int {} // expected-note {{the function body will never be executed}}
```

Comment:
This doesn't seem expected. It seems like it should only be a warning if you've put code in the body of the function.

Reply:
Hmm, seems like I need to check if the closure's body is empty or not before I emit a diagnostic note. I'll fix it now!